mystory/tests.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+
+from mystory.models import Mystory
+
+
+TEMP_DATE = '2006-10-25 14:30:59'
+TEMP_STORY = "This is my story"
+
+
+# @pytest.mark.django_db()
+# def test_new_item():
+# item = Mystory(story_date=TEMP_DATE, story_text=TEMP_STORY)
+# item.save()
+
+# assert item.story_text == TEMP_STORY
+
+
+@pytest.fixture
+def astory():
+    return Mystory(story_date=TEMP_DATE, story_text=TEMP_STORY)
+
+
+class TestMystoryModel:
+    def test_new_story(self, astory):
+        assert astory.story_date == TEMP_DATE
+        assert astory.story_text == TEMP_STORY
```

Comment:
you need to add the @pytest.mark.django_db annotation before the class so pytest tests this with the django sql system 
but for that you need to create the model with save
you can take a look at my pr where i made a test to the house creation #39

Reply:
Done